### PR TITLE
Moved Documentation: colors, shapes, errors and contributing guide

### LINKED ISSUE #159: Moving Documentation: colors, shapes, errors and contributing guide
# Story

## Description

As an engineer, I would like to move documentation to the new dev site.

## Motivation

So that all documentation is in a single spot for consumers.

## Definition of Done

We'll know we're done when the following files are moved over and updated where necessary:

- [Shapes, Colors, Locale and Errors](https://github.com/cerner/terra-graphs/blob/bf4630358b20ec203017fef04ff0543844d7227b/packages/terra-graphs-docs/docs/README.md) (As separate pages)
- [Custom Shapes](https://github.com/cerner/terra-graphs/blob/bf4630358b20ec203017fef04ff0543844d7227b/packages/terra-graphs-docs/docs/core/Shape.md)
- [Contributing Guide](https://github.com/cerner/terra-graphs/blob/7b84660ad711ec66d03fe3d0e7eaca0756735a54/packages/terra-graphs-docs/docs/contributing/README.md)

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Closes #159 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

**Review by tbiethman — APPROVED** [review]

**Comment by AshishMotanamGurunadham** [comment]
Looks like there are some broken images on the dev site(in colors page).
image

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Documentation/Contributing/ContributingGuide.1.doc.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    -   Screenshots of the working prototype.
+    -   Files you want the reviewer to focus on.
+-   Respond to all comments.
+-   Do **not** rebase with main and force push until you reach `ready to merge` status.
+
+### Release
+
+Release is automated, and will be performed when PR is merged to `main` branch.
+
+Terra graphs will be released every Tuesed provided there are code changes to be released.
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Documentation/Contributing/ContributingGuide.1.doc.mdx:104]:
```suggestion
Terra graphs will be released every Tuesday provided there are code changes to be released.
```

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Documentation/Contributing/ContributingGuide.1.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+```text
+<Tag>: <Short description> (fixes #1234)
+```
+**Note**:
+1234 in the above example is the [Github issue number](https://github.com/cerner/terra-graphs/issues).
+
+#### Pull Request
+
+-   Provide the PR with all necessary information to ease code review process in the `Description`, for instance:
+    -   Add the appriopiate tags such as `bug-fix` or `enhancement-fix`.
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Documentation/Contributing/ContributingGuide.1.doc.mdx:93]:
```suggestion
    -   Add the appropriate tags such as `bug-fix` or `enhancement-fix`.
```

**Comment by sdadn** [comment]
@AshishMotanamGurunadham Updated the color hyperlinks and they seem to be working again:
CleanShot 2021-08-13 at 16 58 35@2x

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Pull request merged**